package.json

```diff
@@ -24,18 +24,18 @@
   },
   "dependencies": {
     "ansi": "^0.3.1",
-    "bplist-parser": "^0.1.0",
+    "bplist-parser": "^0.2.0",
     "cross-spawn": "^6.0.5",
```

Comment:
There already is `cross-spawn@7`, but if we remove it in #98 this will not matter anymore

Reply:
👍